lib/app-express.js

```diff
@@ -1,11 +1,32 @@
 const oauth2 = require('salesforce-oauth2');
 const jsforce = require('jsforce');
 const postgresHelper = require('./postgres.js');
+const memjs = require('memjs').Client;
+
+const mjs = memjs.create();
 
 const callbackUrl = process.env.CALLBACKURL;
 const consumerKey = process.env.CONSUMERKEY;
 const consumerSecret = process.env.CONSUMERSECRET;
 
+const allowHostedOneClick = process.env.HOSTED_ONE_CLICK || false; // default for original flavor.
+let conn;
+
+// did this app or any of its peers already git init'd?  If so, there'll be a cached value for that
+if (allowHostedOneClick){
+  mjs.get('conn', (err, cache) => {
+    if (err){
+      console.log(err);
```

Comment:
We should include details like `console.log('Error getting cache info', err);`.

Reply:
agreed, done.